### PR TITLE
Add standalone API server frontend

### PR DESCRIPTION
**Pull request by jmccance**
Add a Ktor-based HTTP frontend to the project that can be deployed as a
Docker image.

### CODE REVIEW

**Comment by bhushanRamnani** [comment]
The previous commit introduced a bug, where the report is getting base64 encoded twice. We had this earlier, which was fixed by https://github.com/Co-Epi/coepi-backend-aws/commit/2ae8d37d9dc5ed70c31c18ff2569d436ccb00017

This is because jackson objectmapper base64 encodes binary data by default.

```
f0189818d391:terraform ramnanib$ curl -X POST https://fls3n3ic7e.execute-api.us-west-1.amazonaws.com/v4/tcnreport -d "ZXlKMFpYTjBJam9pWW05a2VTSjk="
f0189818d391:terraform ramnanib$ curl -X GET https://fls3n3ic7e.execute-api.us-west-1.amazonaws.com/v4/tcnreport
["WlhsS01GcFlUakJKYW05cFdXMDVhMlZUU2prPQ=="]f0189818d391:terraform ramnanib$ 
f0189818d391:terraform ramnanib$ 
f0189818d391:terraform ramnanib$ base64 -D WlhsS01GcFlUakJKYW05cFdXMDVhMlZUU2prPQ==
Unable to open 'WlhsS01GcFlUakJKYW05cFdXMDVhMlZUU2prPQ==': No such file or directory
f0189818d391:terraform ramnanib$ echo "WlhsS01GcFlUakJKYW05cFdXMDVhMlZUU2prPQ==" | base64 -D
ZXlKMFpYTjBJam9pWW05a2VTSjk=
```

File: api-server/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+coepi-backend-aws/api-server
```
Comment (bhushanRamnani) [inline api-server/README.md:1]: In the base README, it'd be good to mention that there are two options of running and deploying this server with a link to this readme for API server.

File: api-server/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+A standalone frontend for the CoEpi Cloud API, suitable for deploying to Kubernetes or any other containerized solution.
+
+## Getting Started
+
+The API server can be run locally from Gradle. Note you will need to have AWS configured correctly so that the service can access the DynamoDB table from your local machine.
+
+```shell script
+./gradlew api-server:run
+```
+
```
Comment (bhushanRamnani) [inline api-server/README.md:13]: Please provide a Get and Post example with localhost.

**Comment by jmccance** [comment]
Created https://github.com/Co-Epi/coepi-backend-aws/pull/31 for the double-encoding bug.

**Comment by jmccance** [comment]
@bhushanRamnani Update docs.

**Pull request merged**